templates/server.conf.j2

```diff
@@ -13,11 +13,19 @@ server-bridge nogw
 
 dev tap
 {% else %}
-server {{openvpn_tunnel_network}} {{openvpn_tunnel_netmask}} nopool
-ifconfig-pool {{openvpn_tunnel_dhcp_range_start}} {{openvpn_tunnel_dhcp_range_end}}
-    {%- if openvpn_tunnel_ipv6_network is defined %}
-
-server-ipv6 {{openvpn_tunnel_ipv6_network}}
+server {{ openvpn_tunnel_subnetv4 | ipaddr('network') }} {{ openvpn_tunnel_subnetv4 | ipaddr('netmask') }} nopool
+{% set ipv4_range_start = openvpn_tunnel_subnetv4 | ipaddr(openvpn_tunnel_dynamic_ipv4_range_start | ipaddr('int')) %}
+{% set ipv4_range_start = (ipv4_range_start | ipaddr('address') if ipv4_range_start) | mandatory("openvpn_tunnel_dynamic_ipv4_range_start '{}' is not in range '{}'".format(openvpn_tunnel_dynamic_ipv4_range_start, openvpn_tunnel_subnetv4)) %}
```

Comment:
Why is the `if` needed? When would `ipv4_range_start` be `None`?

Reply:
For instance, if the range starts at 100, `| ipaddr(100)` outputs False if the input is a subnet with less than 101 IPs.